pkg/chartutil/validate_name.go

```diff
@@ -37,7 +37,7 @@ var validName = regexp.MustCompile(`^[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-
 
 var (
 	// errMissingName indicates that a release (name) was not provided.
-	errMissingName = errors.New("no name provided")
+	errMissingName = errors.New("name is required")
```

Comment:
I would not change this as this is not the error but what is required.

Reply:
Yes, then there is no need to change, it has been restored.